packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -1441,6 +1442,15 @@ class _SelectableFragment with Selectable, Diagnosticable, ChangeNotifier implem
         } else {
           result = _handleSelectParagraph(selectParagraph.globalPosition);
         }
+      case SelectionEventType.selectLine:
+        final SelectLineSelectionEvent selectLine = event as SelectLineSelectionEvent;
+        if (selectLine.absorb) {
```

Comment:
can you explain a bit what absorb is and how is it use?

Reply:
I added absorb to support cases like below:

| Start of paragraph | | inline element | | End of paragraph |

If we click somewhere on the `Start of paragraph` or `End of paragraph` it should select all content in the inline element. This is consistent with the behavior on web and with our `SelectableText`.

https://github.com/user-attachments/assets/c422091b-e79e-4809-ba3f-28dc2b0029e7


https://github.com/user-attachments/assets/32d9ae36-fd6b-4fc3-bc02-37dc0aadd40e

When a `Selectable` receives this event set to true it should select all of its contents, and be absorbed into the boundary. When it is false it should process the event as normal.